supplementary/devices/yuga.rst

```diff
@@ -1,22 +1,22 @@
 
 Yuga - Sony Xperia Z
-=====================
+====================
 
-Here only one or two sentences. What makes the device special? Is it a reference device? 
+There exists basic mainline kernel support added by Sony. This possibly could
+become a reference device.
 
 Status
 ------
 
 Halium
 ^^^^^^
 
-There is a **working Halium 7.1** made by `LNJ2 <https://github.com/LNJ2>`_ and the according Halium/Projectmanagement issue is `here <https://github.com/Halium/projectmanagement/issues/19>`_.
+There is a **working Halium 7.1** made by `LNJ <https://github.com/LNJ2>`_ and
+the according Halium/Projectmanagement issue is `here <https://github.com/Halium/projectmanagement/issues/19>`_.
```

Comment:
When making links, make the link text explanatory. This is for a couple reasons:

* Sphinx does not like it when multiple links are named the same thing. Making links with names like "here" makes this more likely.
* Links should describe their target in general